### PR TITLE
Bump to Quarkus 3.0.1.Final

### PR DESCRIPTION
**Pull request by rsvoboda**
Bump to Quarkus 3.0.1.Final

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -15,4 +15,4 @@ activity.daily-status.repositories.quarkus-extensions-combinations=https://githu
 activity.daily-status.repositories.quarkus-startstop=https://github.com/quarkus-qe/quarkus-startstop/actions/workflows/ci.yaml
 activity.daily-status.repositories.quarkus-test-framework=https://github.com/quarkus-qe/quarkus-test-framework/actions/workflows/daily.yaml
 activity.daily-status.repositories.quarkus-test-suite=https://github.com/quarkus-qe/quarkus-test-suite/actions/workflows/daily.yaml
-activity.daily-status.repositories.quarkus-test-suite-java-17=https://github.com/quarkus-qe/quarkus-jdkspecifics/actions/workflows/daily.yaml
\ No newline at end of file
+activity.daily-status.repositories.quarkus-test-suite-java-17=https://github.com/quarkus-qe/quarkus-jdkspecifics/actions/workflows/ci.yml
```
Comment (michalvavrik) [inline src/main/resources/application.properties:18]: @rsvoboda could you provide context here, please? it sounds awkward that daily status is based on `ci.yml`, what did I miss?

Comment (michalvavrik) [reply src/main/resources/application.properties:18]: ah, I see

Comment (michalvavrik) [reply src/main/resources/application.properties:18]: https://github.com/quarkus-qe/quarkus-jdkspecifics/commit/e0fc4f38501d37f1f3e0997aa49ee01602b40dde

**Review by michalvavrik — APPROVED** [review]
LGTM

File: src/main/resources/application.properties (hunk 1 above)
Comment (rsvoboda) [reply src/main/resources/application.properties:18]: See https://github.com/quarkus-qe/quarkus-jdkspecifics/blob/main/.github/workflows/ci.yml#L3-L6

I've unified the definition for GH Actions

Comment (michalvavrik) [reply src/main/resources/application.properties:18]: sure, thx

**Pull request merged**